### DISCUSSION TITLE
Training on graphs with different sizes

### DISCUSSION THREAD

**Discussion by risufaj** [pyg-team/pytorch_geometric#3334] [Q&A] [answered]
Hi,

I'm pretty new to the field of graph neural networks and pytorch geometric, so this might be quite obvious. However, is it possible to train a model on a dataset of differently sized graphs (e.g. each graph has a different number of nodes). 

Let's assume that I want to make graph level predictions, would this be possible?

Thank you in advance.

**Comment by rusty1s** [comment, marked as answer]
Yes, this is possible and supported out-of-the-box via our `DataLoader`, see [here](https://pytorch-geometric.readthedocs.io/en/latest/notes/introduction.html#mini-batches). [Here](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/mutag_gin.py) is an example on how to use.

**Reply by risufaj** [reply]
Brilliant, thank you very much.
